### DISCUSSION TITLE
Inquiry on DXF to IFC Conversion using ifcopenshell

### DISCUSSION THREAD

**Discussion by AbdelazizBabioui** [IfcOpenShell/IfcOpenShell#4293] [Q&A]
I have been exploring the capabilities of ifcopenshell , and I am interested in the possibility of converting a DXF (or a DWG) file (2D CAD) into an IFC file using ifcopenshell , especially if it was possible to write a script to do the task.

I need to use IfcOpenShell-Python (the ifcopenshell python module) , but i have no idea where to start from .

**Comment by Moult** [comment]
https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.7.0/src/blenderbim/scripts/dxf2ifc.py this might be a good starting point
